inference-engine/samples/benchmark_app/README.md

```diff
@@ -93,6 +93,7 @@ Options:
     -progress                 Optional. Show progress bar (can affect performance measurement). Default values is "false".
     -shape                    Optional. Set shape for input. For example, "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]" in case of one input size.
     -layout                   Optional. Prompts how network layouts should be treated by application. For example, "input1[NCHW],input2[NC]" or "[NCHW]" in case of one input size.
+    -use_remote_mem           Optional. Prealloc remote memory in device to execute infer request.
```

Comment:
I suppose it would be better to add "VPUX-specific performance options" section and put this one here

Reply:
Updated, thanks.